drivers/tripplite_usb.c

```diff
@@ -331,7 +331,7 @@ static int reconnect_ups(void)
 	upsdebugx(2, "= device has been disconnected, try to reconnect =");
 	upsdebugx(2, "==================================================");
 
-	ret = comm_driver->open_dev(&udev, &curDevice, reopen_matcher, NULL);
+	ret = comm_driver->open_dev(&udev, &curDevice, reopen_matcher, match_ups_id);
```

Comment:
You mention the method in reconnect, but declare it later - either move the definition even further up, or add a prototype early in the file (with one, it probably will not like being "static" anymore - similar like when you have prototypes in a header first).